src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
In order to create the swagger.json you need SWG\Info, so it has to be somewhere. I could stick it at the top of a class somewhere but there's not really an appropriate class